src/AppBundle/Entity/Location.php

```diff
@@ -216,4 +238,38 @@ public function getAddress()
     {
         return $this->address;
     }
+
+    /**
+     * Add voteSetting
+     *
+     * @param \AppBundle\Entity\VoteSettings $voteSetting
+     *
+     * @return Location
+     */
+    public function addVoteSetting(\AppBundle\Entity\VoteSettings $voteSetting)
+    {
+        $this->voteSetting[] = $voteSetting;
```

Comment:
I can't understand  what you want to say. I want to tell you only that: check if the item exist before adding or removing. For example: if (!$this->getVoteSetting()->contains($voteSetting)) {
            $this->getVoteSetting()->add($voteSetting);
        }    For my mind it is obviously.